.github/workflows/release.yml

```
@@ -1,12 +1,8 @@
 name: Release
 on:
   pull_request:
-    types: [closed]
-    branches:
-      - main
-
-permissions:
-  contents: read # for checkout
+    branches: [ main ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)